docs/guides/troubleshoot/troubleshooting-agent-with-cloud-connection.md

```diff
@@ -0,0 +1,128 @@
+- [Network issues while connecting to the Cloud](#network-issues-while-connecting-to-the-cloud)
+
+## The claiming script failed
+
+### Make sure the Agent is running
+
+Check if the Agent is running:
+
+```bash
+systemctl status netdata
+```
+
+The expected output should contain info like this:
+
+```bash
+Active: active (running) since Wed 2022-07-06 12:25:02 EEST; 1h 40min ago
+```
+
+:::note
+
+The Agent must be running for the claiming to work.
+
+:::
+
+If the Agent is not running, enable the service:
+
+```bash
+sudo systemctl enable netdata 
+```
+
+and then start the Agent:
+
+```bash
+sudo systemctl start netdata 
+```
+
+:::info
+
+Read more about [Starting, Stopping and Restarting the Agent](https://learn.netdata.cloud/docs/configure/start-stop-restart).
+
+:::
+
+## Claiming on an older, deprecated version of the Agent
+
+Make sure that you are using the latest version of Netdata if you are using the [Claiming script](https://learn.netdata.cloud/docs/agent/claim#claiming-script).
+
+With the introduction of our new architecture, Agents running versions lower than `v1.32.0` can face claiming problems, so we reccomend to [update the Netdata Agent](https://learn.netdata.cloud/docs/agent/packaging/installer/update).
+
+## Network issues while connecting to the Cloud
+
+### Check your IP
+
+It is possible that your IP might be banned from `app.netdata.cloud`, for security reasons.
+
+To check this, run:
+
+```bash
+sudo netdatacli aclk-state 
+```
+
+The output will contain a line indicating if the IP is banned from `app.netdata.cloud`:
+
+```bash
+Banned By Cloud: No
+```
+
+If your node's IP is banned, you can:
+
+- Contact our team to whitelist your IP by sumbiting a ticket at [Netdata Community](https://community.netdata.cloud/)
+- Change your node's IP
+
+### Make sure that you have an internet connection
+
+Firstly, check that you have internet connection by pinging well known hosts:
+
+```bash
+ping 8.8.8.8
+```
+
+or
+
+```bash
+ping 1.1.1.1
+```
+
+:::tip
+
+Exit the `ping` command by pressing `Ctrl+C`
+
+:::
+
+### Check DNS resolution
+
+You can check your DNS resolution by running:
+
+```bash
+host app.netdata.cloud
+```
+
+The expected output should be something like this:
+
+```bash
+app.netdata.cloud is an alias for main-ingress-545609a41fcaf5d6.elb.us-east-1.amazonaws.com.
+main-ingress-545609a41fcaf5d6.elb.us-east-1.amazonaws.com has address 54.198.178.11
+main-ingress-545609a41fcaf5d6.elb.us-east-1.amazonaws.com has address 44.207.131.212
+main-ingress-545609a41fcaf5d6.elb.us-east-1.amazonaws.com has address 44.196.50.41
+```
+
+If the command fails, you can whitelist the `app.netdata.cloud` domain from the node's firewall restrictions.
+
+:::tip
+
+If you can't whitelist domains from your firewall, you can whitelist the IPs that the above mentioned command will produce, but keep in mind that they can change at anytime without notice.
+
+:::
```

Comment:
I think the certificate issue might be another one worth noting here - I've seen multiple users run into this


https://community.netdata.cloud/t/certificate-verification-error-connecting-to-the-cloud/1790

 https://community.netdata.cloud/t/claimed-nodes-not-reporting-to-cloud/1282/7

Reply:
We should add a section at the end which says if none of these steps worked then the user should reach out to us via discord/forum/github etc.

Reply:
And finally, we should let the SREs do a quick review of this as well to ensure it's technically accurate.

Reply:
Agree on the discord/forum/github guidance!

Reply:
We can tag the SRE team, @shyamvalsan , @Ancairon to do a quick read through of this.